.gitignore

```
@@ -1,3 +1,3 @@
 .idea/*
 *.iml
-target/*
+target/*
```

add black line in the end
```suggestion
target/*

```

---

It should be 4 lines? 3 with text and one blank?